sources/test/test_ostree_source.py

```diff
@@ -42,7 +44,9 @@ def make_test_sources(proto, port, fake_commit, **secrets):
 
 def make_repo(root):
     with tempfile.TemporaryDirectory() as empty_tmpdir:
```

Comment:
(super nitpick) the tempdir here is no longer empty after the change, i.e. now "a.txt" and "b.txt" are added